src/server.ts

```diff
@@ -62,7 +63,7 @@ import { GOVERNANCE_URL } from './constants'
 const jobs = manager()
 jobs.cron('@eachMinute', finishProposal)
 jobs.cron('@eachMinute', activateProposals)
-jobs.cron('@eachMinute', publishBids)
+jobs.cron('@eachMinute', withLock('publishBids', publishBids))
```

Comment:
Would it not be beneficial to also increase the frequency with which it runs? Maybe every 5 minutes for example